rbd-target-api.py

```diff
@@ -1069,17 +1069,26 @@ def disk(pool, image):
         if disk_usable != 'ok':
             return jsonify(message=disk_usable), 400
 
-        create_image = request.form.get('create_image') == 'true'
-        if mode == 'create' and (not create_image or not size):
+        create_image = request.form.get('create_image', 'true')
+        if create_image not in ['true', 'false']:
+            logger.error("Invalid 'create_image' value {}".format(create_image))
+            return jsonify(message="Invalid 'create_image' value {}".format(create_image)), 400
+
+        if mode == 'create' and (create_image == 'false' or not size):
             try:
+                # not size meaning not intend to creat an image, try to
```

Comment:
`not size meaning not intend to creat an image, try to` -> `no size implies not intention to create an image, try to

Reply:
Done.